python/src/scipp/plot/resampling_model.py

```diff
@@ -63,14 +63,19 @@ def _rebin(self, var, coords):
 
     def _make_edges(self, params):
         edges = []
-        for dim, par in params.items():
+        for i, (dim, par) in enumerate(params.items()):
```

Comment:
Couldn't edges just be a dictionary where you can store `res` as key and Variable as value? Then you can just resturn `sorted(edges)`

Reply:
this doesn't work if you have duplicate `res`... but it would be good to use sorted somehow... as it may be good to always `rebin` the small dims first?